babel.config.js

```diff
@@ -17,32 +18,23 @@ module.exports = function generateConfig(api) {
     ],
     ignore: ['__mocks__'],
     plugins: [
+      ['transform-react-remove-prop-types', { removeImport: true }],
```

Comment:
We need to wrap the prop types, not strip them. If we strip them, applications consuming this library won't get prop type warnings when using our components.

We should use [`mode: 'unsafe-wrap'`](https://www.npmjs.com/package/babel-plugin-transform-react-remove-prop-types#mode):
> The wrap modes are targeting React libraries like material-ui or react-native-web. They are not intended to be used by application authors.

[See the material-ui babel config here](https://github.com/mui-org/material-ui/blob/master/babel.config.js#L34-L40)

```suggestion
      ['transform-react-remove-prop-types', { removeImport: true, mode: 'unsafe-wrap', }],
```

Reply:
this was used in new solution as well.